frontend/src/App.tsx

```diff
@@ -120,8 +123,11 @@ class App extends Component<Record<string, never>> {
 						onDecline={
 							() => { gaSetState(true); window.location.reload(); }
 						}
+						// Due to the version of react-cookie-consent used missing the type declaration for the "visible" prop, this hack is sadly required.
+						/// @ts-expect-error
```

Comment:
Linter error needs fixing (see error attached in the review panel)

Reply:
file has been updated